contracts/Governor/GovernorModule.sol

```diff
@@ -53,6 +53,8 @@ contract GovernorModule is
         __GovernorTimelock_init(_timelock);
         __initBase(_accessControl, msg.sender, "Governor Module");
         __GovernorPreventLateQuorum_init(_initialVoteExtension);
+        _registerInterface(type(IGovernorModule).interfaceId);
+        _registerInterface(type(IGovernorTimelock).interfaceId);
```

Comment:
Ok very nice.

While we're at it, are there any other contracts in this repo where it makes sense to do this?

I ask because the inheritance tree in here is honestly still kind of confusing to me lol

Reply:
To do what specifically? Make abstract contracts be interfaces, or add in more _registerInterfaces?

Reply:
Add more `_registerIntrrfaces`.

Reply:
Hmm yeah, there is a lot of inheritance going on in this codebase, and there may be some abstract contracts that we want ERC-165 supported interfaces for. I'll need to dig into the codebase a bit more to figure out what we need

Reply:
Are you going to do that digging as a part of this PR, or in the future?

Reply:
I'll do it in this PR. Looking now into which of these abstract OZ contracts have interfaces in the OZ repo, and planning on registering these interfaces in the initializer.